Cargo.lock

```diff
@@ -4513,7 +4760,16 @@ version = "1.0.61"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c546c80d6be4bc6a00c0f01730c08df82eaa7a7a61f11d656526506112cc1709"
 dependencies = [
- "thiserror-impl",
+ "thiserror-impl 1.0.61",
+]
+
+[[package]]
+name = "thiserror"
+version = "2.0.11"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "d452f284b73e6d76dd36758a0c8684b1d5be31f92b89d07fd5822175732206fc"
+dependencies = [
+ "thiserror-impl 2.0.11",
```

Comment:
note: Seems that hyper & friends are using a new major release of `thiserror`. Our crates and some other dependencies still use 1.0, so 2 versions are pulled, but because `thiserror` basically does macros, it shouldn't have major codegen implications.